server/src/main/java/io/kroki/server/Server.java

```diff
@@ -178,16 +178,29 @@ static void start(Vertx vertx, JsonObject config, Handler<AsyncResult<HttpServer
 
   private static void setPemKeyCertOptions(JsonObject config, HttpServerOptions serverOptions, boolean enableSSL) {
     if (enableSSL) {
-      Optional<String> sslKeyValue = Optional.ofNullable(config.getString("KROKI_SSL_KEY"));
-      Optional<String> sslCertValue = Optional.ofNullable(config.getString("KROKI_SSL_CERT"));
-      if (sslKeyValue.isEmpty() || sslCertValue.isEmpty()) {
-        throw new IllegalArgumentException("KROKI_SSL_KEY and KROKI_SSL_CERT must be configured when SSL is enabled.");
+      PemKeyCertOptions certOptions = new PemKeyCertOptions();
+
+      if (!config.containsKey("KROKI_SSL_KEY") && !config.containsKey("KROKI_SSL_KEY_PATH")) {
+        throw new IllegalArgumentException("KROKI_SSL_KEY or KROKI_SSL_KEY_PATH must be configured when SSL is enabled.");
       }
-      serverOptions.setPemKeyCertOptions(
-        new PemKeyCertOptions()
-          .addKeyValue(Buffer.buffer(sslKeyValue.get()))
-          .addCertValue(Buffer.buffer(sslCertValue.get()))
-      );
+      else if (config.containsKey("KROKI_SSL_KEY") && !config.getString("KROKI_SSL_KEY").isEmpty()) {
```

Comment:
we don't need `else if` when the `if` block returns a value or throws an error

```suggestion
      if (config.containsKey("KROKI_SSL_KEY") && !config.getString("KROKI_SSL_KEY").isEmpty()) {
```